scripts/ci-e2e.sh

```diff
@@ -10,6 +10,22 @@ FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
 
 export CAPM3RELEASEBRANCH="${CAPM3RELEASEBRANCH:-main}"
 
+# Extract release version from release-branch name
+if [[ "${CAPM3RELEASEBRANCH}" == release-* ]]; then
+  CAPM3_RELEASE_PREFIX="${CAPM3RELEASEBRANCH#release-}"
+else
+  CAPM3_RELEASE_PREFIX=""
+fi
+
+# Fetch CAPI version that coresponds to CAPM3_RELEASE_PREFIX release version
+if [[ "${CAPM3_RELEASE_PREFIX}" =~ ^(1\.6|1\.7|1\.8|1\.9)$ ]]; then
+  export CAPM3RELEASE="v${CAPM3_RELEASE_PREFIX}.99"
+  export CAPI_RELEASE_PREFIX="v${CAPM3_RELEASE_PREFIX}."
+else
+  export CAPM3RELEASE="v1.10.99"
```

Comment:
you need to add a patch in dev-env also because this is not hard coded in dev-env

Reply:
Its here: https://github.com/metal3-io/metal3-dev-env/pull/1487

Reply:
Ah great